internal/config/config.go

```diff
@@ -1,9 +1,43 @@
 package config
 
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"path"
+)
+
 // Config holds information about how the peregrine backend is configured.
 type Config struct {
 	Server struct {
 		Address string `json:"address"`
 		Origin  string `json:"origin"`
 	} `json:"server"`
+
+	TBA struct {
+		URL    string `json:"URL"`
+		APIKey string `json:"apiKey"`
+	} `json:"tba"`
+
+	Database struct {
```

Comment:
If you made this `store.Options` (after adding json tags to it) you could just pass it straight to `store.New`